lib/cn/blk_list.c

```
@@ -51,7 +51,7 @@ commit_mblocks(struct mpool *mp, struct blk_list *blks)
     return 0;
 }
 
-merr_t
+void
```

Any particular reason you don't want to propagate the error to the caller?